<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="2" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-01-09T13:01:08Z" name="Default suite" finished-at="2021-01-09T13:01:22Z" duration-ms="14222">
    <groups>
    </groups>
    <test started-at="2021-01-09T13:01:08Z" name="Default test" finished-at="2021-01-09T13:01:22Z" duration-ms="14222">
      <class name="com.w2a.testcases.Login">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:05Z" name="setUp" finished-at="2021-01-09T18:31:08Z" duration-ms="2769" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method is-config="true" signature="startTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:08Z" name="startTest" finished-at="2021-01-09T18:31:08Z" duration-ms="51" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method signature="FailTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:08Z" name="FailTest" finished-at="2021-01-09T18:31:08Z" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FailTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:08Z" name="tearDown" finished-at="2021-01-09T18:31:08Z" duration-ms="1" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=FailTest status=SUCCESS method=Login.FailTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="startTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:08Z" name="startTest" finished-at="2021-01-09T18:31:08Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startTest -->
        <test-method signature="LoginTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:08Z" name="LoginTest" finished-at="2021-01-09T18:31:22Z" duration-ms="14152" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:22Z" name="tearDown" finished-at="2021-01-09T18:31:22Z" duration-ms="0" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=SUCCESS method=Login.LoginTest()[pri:0, instance:com.w2a.testcases.Login@2a693f59] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.w2a.testcases.Login@2a693f59]" started-at="2021-01-09T18:31:22Z" name="tearDown" finished-at="2021-01-09T18:31:22Z" duration-ms="0" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "base.TestBase.extent" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "com.aventstack.extentreports.ExtentReports.flush()" because "base.TestBase.extent" is null
	at base.TestBase.tearDown(TestBase.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.w2a.testcases.Login -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
